### DISCUSSION TITLE
Is there anyway we can name the release using `kamal deploy` and rollback?

### DISCUSSION THREAD

**Discussion by darkamenosa** [basecamp/kamal#552] [Q&A]
I used kamal to deploy successfully. But I have too many deployments, I have no idea which one to select to roll back when having issue.

Is there any way we can add a name to our image every `kamal deploy`?

**Comment by acidtib** [comment]
Kamal tags the images with a hash of the current commit in the repository's history, https://github.com/basecamp/kamal/blob/aea55480adcaf61e3eebfd49f2b8c039207ad0de/lib/kamal/configuration.rb#L302

for example, kamal tags an image
`ghcr.io/acidtib/the-plantain:5195d4dd2788708355e2e63747ea19306219faf4`

i can now use that hash to deploy that image
```
kamal deploy --skip-push --version=5195d4dd2788708355e2e63747ea19306219faf4
```

or rollback to it
```
kamal rollback 5195d4dd2788708355e2e63747ea19306219faf4
```
